pkgs/data/fonts/noto-fonts/default.nix

```diff
@@ -61,6 +60,43 @@ let
         maintainers = with maintainers; [ mathnerd314 emily ];
       };
     };
+
+  mkNotoCJK = { typeface, version }:
+    stdenvNoCC.mkDerivation {
+      pname = "noto-fonts-cjk-${lib.toLower typeface}";
+      inherit version;
+
+      src = fetchFromGitHub {
+        owner = "googlefonts";
+        repo = "noto-cjk";
+        rev = "9f7f3c38eab63e1d1fddd8d50937fe4f1eacdb1d";
+        sha256 = "sha256-BX4tcDcz+RGka8mtced1k3BopUJQ14t1BtAVqTjyPik=";
+      };
+
+      installPhase = ''
+        install -m444 -Dt $out/share/fonts/opentype/noto-cjk ${typeface}/Variable/OTC/*.otf.ttc
+      '';
+
+      meta = with lib; {
+        description = "Beautiful and free fonts for CJK languages";
+        homepage = "https://www.google.com/get/noto/help/cjk/";
+        longDescription =
+        ''
+          Noto Sans CJK is a sans serif typeface designed as an intermediate style
```

Comment:
This says Sans, I guess you should remove that or change it to `${typeface}`

Reply:
I changed it to `${typeface}`.